docs/en/stack/ml/anomaly-detection/ml-troubleshooting.asciidoc

```diff
@@ -164,20 +167,17 @@ PUT .ml-config
   "settings": { "auto_expand_replicas": "0-1"}
 }
 
-# 6. Reindex the temporary index into the .ml-config index
+# 6. Disable upgrade mode
+POST _ml/set_upgrade_mode?enabled=false&timeout=10m
```

Comment:
For the config index upgrade mode should stay in force until after reindex.  It's different for the annotations index as they're used at different times and created in different ways.

Reply:
Done, thanks!